helm/default-apps-aws/values.yaml

```diff
@@ -122,6 +122,17 @@ apps:
     # used by renovate
     # repo: giantswarm/cert-manager-app
     version: 3.4.0
+  chartOperatorExtensions:
+    appName: chart-operator-extensions
+    chartName: chart-operator-extensions
+    catalog: default
+    enabled: true
+    clusterValues:
+      configMap: true
+      secret: true
```

Comment:
Does it need to use the configmap and secret cluster values?

Reply:
Not really, but for one the admission controller will mutate it on anyway I think + for the PSP/PSS migration it is all for having some shared values. I was like it does not hurt to have them on either. I remove them tho if you say it is fine 👍🏻

Reply:
Updates 👍🏻